test/evergreen.yml

```diff
@@ -136,8 +145,9 @@ functions:
       script: |
         set -o errexit
         set -o verbose
+        echo "generated_token: ${generated_token}"
```

Comment:
leftover?

Reply:
Good catch. Removed in [ef6f023](https://github.com/wiredtiger/wiredtiger/pull/11068/commits/ef6f023be18a7a0543bbdfa00922e945db3a113a)